utils/servo/LinearServo.java

```diff
@@ -1,49 +0,0 @@
-package frc.team670.mustanglib.utils.servo;
-
-import edu.wpi.first.wpilibj.Servo;
-
-/**
- * Interfaces with a Linear Servo through PWM
- * 
- * @author AkshatAdsule
- */
-public class LinearServo {
```

Comment:
I'm curious why we're deleting this. We haven't used a servo recently but we very well might. Is there something wrong with this logic?